types/materializecss__materialize/index.d.ts

```diff
@@ -0,0 +1,40 @@
+// Type definitions for materializecss__materialize 1.1
+// Project: https://materializecss.github.io/materialize/
```

Comment:
```suggestion
// Project: https://materializecss.github.io/materialize/
```
so if this is a new definition for a fork, the project here is obviously wrong and can trick people

Reply:
skip this one, I can't read in the dark :)